### PR TITLE
Use admin status to determine navbar links

### PR DESCRIPTION
**Pull request by dfaulken**
More progress on #35.

Next steps: locking down doing things to assignments.

As admin in current roster:

screenshot from 2017-04-17 14-05-56

As admin, but in another roster:

screenshot from 2017-04-17 14-06-12

As non-admin:

screenshot from 2017-04-17 14-06-24

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -21,4 +21,12 @@ def full_name
   def proper_name
     "#{last_name}, #{first_name}"
   end
+
+  def admin_in?(roster)
+    memberships.find_by(roster: roster).admin?
```
Comment (Anbranin) [inline app/models/user.rb]: I guess the only thing fishy about having being an admin reliant on there being an existing roster is that any new users that want to use this service will need to have an existing admin create a roster for him and make him an admin of that. Or go through IT.

File: app/models/user.rb [hunk 2]
```diff
@@ -21,4 +21,12 @@ def full_name
...
+  end
+
+  def admin?
```
Comment (Anbranin) [inline app/models/user.rb]: The only issue I can see with having an 'admin' status relying on the fact that you are an admin of an existing roster would be that any new potential administrators of new rosters would have to go through an existing admin to create their roster for them.

Comment (dfaulken) [inline app/models/user.rb]: Okay. What alternative workflow might you suggest?

Comment (Anbranin) [inline app/models/user.rb]: One alternative is having a regular old 'administrator' column on a user itself, and then allow current administrators to edit the users to make them admins.. but.. this seems kind of awful, so carry on.

**Pull request merged**